recovery-init

```diff
@@ -5,6 +5,7 @@
 # This file is part of Jolla recovery console
 #
 # Copyright (C) 2013-2015 Jolla Ltd.
+# Copyright (c) 2021. Open Mobile Platform LLC.
```

Comment:
No full-stop after the date, see e.g. https://github.com/sailfishos/sailjail/blob/master/daemon/test/test_appinfo.c#L2
```suggestion
# Copyright (c) 2021 Open Mobile Platform LLC.
```